File: go/tcso.go
package main

import (
	"fmt"
	"errors"
	"runtime"
	"os/exec"
)


func main() {
   	//write the code which can return error
    if err != nil {
        //open stack overflow with go error
		var strs string
		err := errors.New(err.Error())
		strs="http://stackoverflow.com/search?q=[go]"+err.Error()
        openbrowser(strs)
    }
}

func openbrowser(url string) {
	var err error
	err = errors.New("an error message")
	switch runtime.GOOS {
	case "linux":
		err = exec.Command("xdg-open", url).Start()
	case "windows":
		err = exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Start()
	case "darwin":
		err = exec.Command("open", url).Start()
	default:
		err = fmt.Errorf("unsupported platform")
	}
	if err != nil {
		fmt.Printf(err.Error())
	}
}

File: elixir/tcso.ex
defmodule OpenBrowser do
  def open(url) do
    IO.puts(url)

    start_browser_command =
      case :os.type() do
        {:win32, _} ->
          "explorer"

        {:unix, :darwin} ->
          "open"

        {:unix, _} ->
          "xdg-open"
      end

    if System.find_executable(start_browser_command) do
      System.cmd(start_browser_command, [url])
    else
      Mix.raise("Command not found: #{start_browser_command}")
    end
  end
end

defmodule TcSo do
  try do
    # Do something bad :-)
    exit("I am exiting")
  catch
    e ->
      url = "http://stackoverflow.com/search?q=[elixir]+"
      error = e
      OpenBrowser.open(url <> error)
  end
end
